api/chalicelib/core/health.py

```diff
@@ -147,10 +147,7 @@ def __check_SSL(*_):
         }
     }
     try:
-        if(config("is_dns_public")==True):
-            requests.get(config("SITE_URL"), verify=True, allow_redirects=True)
-        else:
-            requests.get(config("SITE_URL"), verify=False, allow_redirects=True)
+        requests.get(config("SITE_URL"), verify=True, allow_redirects=True)
```

Comment:
@SavinienBarbotaud why do we need this block?

Reply:
I delete this block. There is no changes between the start of the branch and this point.